src/main/java/com/troubleshooting/model/IEnvModel.java

```diff
@@ -0,0 +1,6 @@
+package com.troubleshooting.model;
+
+public interface IEnvModel {
```

Comment:
Starting an interface name with a capital I is a C# convention.
an interface should have only methods.